### PR TITLE
fix pass vyper code

### PR DESCRIPTION
**Pull request by yudetamago**

### CODE REVIEW

File: app/src/App.js [hunk 1]
```diff
@@ -33,23 +33,20 @@ class App extends Component {
...
+      plugin.result.placeholderText = result[0]
       extension.call('editor', 'getFile', result, (error, result) => {
         console.log(result)
-        plugin.setState({
-          vyper: result[0]
-        })
-        console.log(plugin.state.vyper)
-        plugin.compile(plugin.onCompileSucceeded, plugin.onCompileFailed)
+        plugin.result.vyper = result[0]
+        plugin.setState(plugin.result)
```
Comment (nrryuya) [inline app/src/App.js]: What does this code mean?

Comment (nrryuya) [reply app/src/App.js]: L42 is enough?

Comment (nrryuya) [reply app/src/App.js]: ah~~~

Comment (nrryuya) [reply app/src/App.js]: I misunderstood
sorry

Comment (yudetamago) [reply app/src/App.js]: `placeholderText` has already been set at L39, then `plugin.result` is like `{placeHolderText: "hoge", vyper: "hoge"}` at L43.
Both state will update.

Comment (yudetamago) [reply app/src/App.js]: no problem!

**Pull request merged**